.github/workflows/build_wheels.yml

```diff
@@ -5,15 +5,16 @@ on:
     branches: [main, master, 'release*']
     tags: ['*']
 
-permissions:
-  contents: write
-
 jobs:
   build-wheels:
     if: github.repository == 'python/mypy'
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
       - uses: actions/checkout@v4
+        with:
+          persist-credentials: true  # needed to push tags
```

Comment:
Not sure the `contents: write` permission is actually needed here. Similarly `persist-credentials` could probably be set to `false`.

The script itself will push a new tag to a different repo (https://github.com/mypyc/mypy_mypyc-wheels) using the `WHEELS_PUSH_TOKEN`. It doesn't effect anything for `python/mypy` itself, thus `read` should be enough.

Reply:
Good catch! 👍

Reply:
Seems you accidentally removed `persist-credentials: false` here. So pre-commit is now complaining.